.github/workflows/ci.yml

```diff
@@ -22,17 +26,41 @@ on:
       - v2.*
       - v3.*
   pull_request:
-    paths-ignore:
-      - "docs/**"
-      - "packages/create-vite/**"
   workflow_dispatch:
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
   cancel-in-progress: true
 
 jobs:
-  build:
+  diff:
+    timeout-minutes: 2
+    runs-on: ubuntu-latest
+    name: "Diff files"
+    outputs:
+      runTest: ${{ steps.skip-tests-files.outputs.only_changed != 'true' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          # Assume PRs are less than 50 commits
+          fetch-depth: 50
+
+      - name: Get changed files
+        id: skip-tests-files
+        uses: tj-actions/changed-files@1d1287f9fafd92be283f99b781fb5f00f00dd471 # v35.2.1
+        with:
+          files: |
+            docs/**
+            .github/**
+            !.github/workflows/ci.yml
+            packages/create-vite/**
+            !packages/create-vite/__tests__/**
+            **/*.md
```

Comment:
I overlooked one thing.
Even if the `packages/create-vite/**` changed, this PR is skipping the unit tests. For example, when only `packages/create-vite/src/index.ts` changed, unit tests won't run with this. I think we should run in that case, too.

Reply:
Good catch!